### PR TITLE
Add meta tags

### PR DESCRIPTION
**Pull request by zuchuandatou**
- Add meta tags for Highwire Press on meta.html
- UI tests for meta tags

### CODE REVIEW

File: tests/ui/test_meta_tags.py
Comment (ekate) [inline tests/ui/test_meta_tags.py:1]: @zuchuandatou, you do not need to add file to the draft record is this test

**Review by ekate — CHANGES_REQUESTED** [review]
@zuchuandatou, please change date on the Copyright statement to include current year.
Also, check my comment about adding file in the test record. i thin k we do not need it in that test

File: tests/ui/test_meta_tags.py
Comment (zuchuandatou) [reply tests/ui/test_meta_tags.py:1]: Thanks for you info! Gonna give it a try without draft record

Comment (ekate) [reply tests/ui/test_meta_tags.py:1]: You need draft record, but do not need to add file to it

Comment (zuchuandatou) [reply tests/ui/test_meta_tags.py:1]: Yeah that's what I mean - used a metadata only record. Just pushed.

**Review by ekate — APPROVED** [review]

**Review by ekate — APPROVED** [review]

**Pull request merged**